flytekit/clis/sdk_in_container/run.py

```diff
@@ -707,6 +708,11 @@ def _run(*args, **kwargs):
 
         console_url = remote.generate_console_url(execution)
         click.secho(f"Go to {console_url} to see execution in the console.")
+        if remote_entity:
+            file = str(remote_entity.id.name).split('.')[0]
+            file += ".py"
```

Comment:
You could use result callback. https://stackoverflow.com/questions/38164324/python-click-having-the-group-execute-code-after-a-command, so that flytekit can remove file after pyflyte run --help

Reply:
filename can be added to click context, like https://github.com/flyteorg/flytekit/blob/4f57ab9009e73aa06a538ec4eafff90443e35219/flytekit/clis/sdk_in_container/run.py#L761-L762, then you can access it in the result callback.